### DISCUSSION TITLE
seasCal failure

### DISCUSSION THREAD

**Discussion by emawbgit** [RobelTakele/AquaBEHER#13] [Q&A] [answered]
Dear Robel,

I have tried to apply your code/command to calculate season onset/cessation. All runs fine, apart from the seasCal that returns the error below.
![image](https://github.com/RobelTakele/AquaBEHER/assets/65594313/3b2770a8-7fbb-48cc-831a-1bb39cb86c5f)

Maybe you could figure out what is the problem?
I attach the R code (
[sos_code.txt](https://github.com/RobelTakele/AquaBEHER/files/13438479/sos_code.txt)
) and dataset ([arable_data.csv](https://github.com/RobelTakele/AquaBEHER/files/13438448/arable_data.csv)) used so that you can replicate the issue.

Thanks in advance and looking forward to your kind feedback,
Emanuele

**Comment by RobelTakele** [comment]
@emawbgit, you need more data that covers at least 1 climatological year. your data starts from 08/23 and ends 11/23. that is not enough.  at least try to get data from 07/23 to 01/24. one more month after the season ends.

**Comment by emawbgit** [comment]
[like]  Emanuele Clemente reacted to your message:
________________________________

**Comment by RobelTakele** [comment, marked as answer]
@emawbgit, check out the new Version (v1.4.0) some of the issues are resolved.
